### PR TITLE
Add labels.json

### PR DESCRIPTION
**Pull request by justinfagnani**

### CODE REVIEW

**Comment by abdonrd** [comment]
A few days ago I create similar labels.
I'll share them with you in case one of them could inspire you...

github-labels

```json
[
  {
    "name": "good first issue",
    "color": "7057ff",
    "description": "Good for newcomers"
  },
  {
    "name": "priority: critical",
    "color": "b60205",
    "description": ""
  },
  {
    "name": "priority: high",
    "color": "d93f0b",
    "description": ""
  },
  {
    "name": "priority: low",
    "color": "0e8a16",
    "description": ""
  },
  {
    "name": "priority: medium",
    "color": "fbca04",
    "description": ""
  },
  {
    "name": "status: can't reproduce",
    "color": "dda035",
    "description": ""
  },
  {
    "name": "status: confirmed",
    "color": "215cea",
    "description": ""
  },
  {
    "name": "status: duplicate",
    "color": "cfd3d7",
    "description": "This issue or pull request already exists"
  },
  {
    "name": "status: have to reproduce",
    "color": "077f53",
    "description": ""
  },
  {
    "name": "status: need more information",
    "color": "fef2c0",
    "description": ""
  },
  {
    "name": "status: wontfix",
    "color": "ffffff",
    "description": "This will not be worked on"
  },
  {
    "name": "type: bug",
    "color": "d73a4a",
    "description": "Something isn't working"
  },
  {
    "name": "type: discussion",
    "color": "d4c5f9",
    "description": ""
  },
  {
    "name": "type: documentation",
    "color": "006b75",
    "description": ""
  },
  {
    "name": "type: enhancement",
    "color": "84b6eb",
    "description": ""
  },
  {
    "name": "type: feature request",
    "color": "fbca04",
    "description": "New feature or request"
  },
  {
    "name": "type: help wanted",
    "color": "008672",
    "description": "Extra attention is needed"
  },
  {
    "name": "type: question",
    "color": "d876e3",
    "description": "Further information is requested"
  }
]
```

File: labels.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+
+[
+  { "name": "Package: analyzer", "color": "#f9df95" },
+  { "name": "Package: build", "color": "#f9df95" },
+  { "name": "Package: bundler", "color": "#f9df95" },
+  { "name": "Package: cli", "color": "#f9df95" },
+  { "name": "Package: project-config", "color": "#f9df95" },
+  { "name": "Package: linter", "color": "#f9df95" },
+  { "name": "Browser: Chrome", "color": "#3477c9" },
```
Comment (rictic) [inline labels.json]: Do we have enough browser or os-specific bugs to be worth making browser and os-specific labels?

File: labels.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  { "name": "Platform: Android", "color": "#bfdadc" },
+  { "name": "Platform: iOS", "color": "#bfdadc" },
+  { "name": "Status: Abandoned", "color": "#000000" },
+  { "name": "Status: Accepted", "color": "#009800" },
+  { "name": "Status: Available", "color": "#bfe5bf" },
+  { "name": "Status: Blocked", "color": "#e11d21" },
+  { "name": "Status: Completed", "color": "#006b75" },
+  { "name": "Status: In Progress", "color": "#cccccc" },
+  { "name": "Status: On Hold", "color": "#e11d21" },
+  { "name": "Status: Pending", "color": "#fef2c0" },
```
Comment (rictic) [inline labels.json]: What's the difference between Pending, On Hold, and Blocked?

File: labels.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  { "name": "Status: Review Needed", "color": "#fbca04" },
+  { "name": "Status: Revision Needed", "color": "#e11d21" },
```
Comment (rictic) [inline labels.json]: Are Review Needed and Revision Needed necessary? They seem like properties of a Pull Request that are already modeled in a first class way by GitHub

File: labels.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
+
```
Comment (rictic) [inline labels.json]: It looks like labels can now have descriptions. Seems worth adding those for the `Status:`, `Priority:`, and `Type:` labels.

**Comment by rictic** [comment]
My reasoning for pushing back on a large number of label kinds is that it adds to the mental overhead when categorizing bugs.

**Comment by justinfagnani** [comment]
This is mostly the set of labels we already use in the CLI and other tools projects, with the addition of the package labels. We've talked about about pruning the *Status* labels, but didn't reach a conclusion. I already sync'ed these labels to the repo, so just wanted to commit the file I used. I think we can revisit them incrementally.

**Comment by rictic** [comment]
If we update `labels.json` will be be able to update the labels in github?

File: labels.json (hunk 1 above)
Comment (justinfagnani) [reply labels.json]: For platform, and in the CLI we definitely have, especially when we were working on initial Windows support. In other repos, the need has occasionally been there, mainly for windows. For Browser we've had lots of browser specific issues.

**Comment by justinfagnani** [comment]
Yes. I'm using git-labelmaker which allows bidirectional syncing: https://github.com/himynameisdave/git-labelmaker

But now that I mention it, the only reason I had been doing this before was to standardize labels across repos, which isn't necessary with a monorepo... 🤔

**Comment by justinfagnani** [comment]
Yeah... closing this as unnecessary

**Pull request closed without merging**